.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Describe the content of the PR and links to related issues, bugs of features.
+
+## Requirements
+
+Before submitting this PR, please make sure:
+
+- [ ] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
+- [ ] The code builds cleanly without new errors or warnings
+- [ ] The code passes all the existing tests
```

Comment:
These shouldn't be needed as part of the checklist, because it is already enforced by Github and our CI system. We already can't merge PRs which don't satisfy these.

I would suggest taking this bullet in particular from the code review checklist, as it's probably the number one easiest and most impactful change authors can include/add to their other changes (of course giving exceptions where we don't have the necessary info, as appropriate).

You could if you prefer combine this bullet with the next one (since they are both about having documentation).
```suggestion
- [ ] Docstrings and type hints are added to new and updated routines, as appropriate
```
```suggestion
- [ ] All relevant documentation has been updated or added. Docstrings and type hints are added to new and updated routines, as appropriate
```